README.md

```diff
@@ -1,14 +1,13 @@
 # openSUSE.org Search
 
-Originally project from 2011, revived now with modern look and feel
-Updated version of search.openSUSE.org federated search engine.
+[Originally created in 2011](https://github.com/openSUSE/searchPage), the project has now been revived with a modern appearance and an updated version at the same address [search.opensuse.org](https://search.opensuse.org).
```

Comment:
That's not the original page

Reply:
> That's not the original page

Is "Originally created" link wrong?

Reply:
Yes, the original website is a branch in this repo, it was hosted on gitorious before